### DISCUSSION TITLE
How to Make a Contributable Obsidian Vault like the Hub

### DISCUSSION THREAD

**Discussion by cybersader** [obsidian-community/obsidian-hub#686] [General]
Is there a way to copy from some of the GitHub actions and tailor it to my own system?  I'm trying to design a contributable wiki system myself and thought initially that Obsidian Publish provided such features until I noticed this was only a part of the .github files (actions, scripts, etc.) in this vault. What I'm really asking is if it would take a long time to or there's caveats and nuance to using some of the work that has already been done in this vault and syncing some of that scripts to my own vault repo?

**Comment by claremacrae** [comment]
Because the Hub was created without a LICENCE, the answer is unfortunately NO, derivative works may not be created:

<https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository>
> You're under no obligation to choose a license. However, without a license, the default copyright laws apply, meaning that you retain all rights to your source code and no one may reproduce, distribute, or create derivative works from your work.
